src/hydrusvideodeduplicator/__main__.py

```diff
@@ -28,6 +28,9 @@ def main(
     api_url: Annotated[Optional[str], typer.Option(help="Hydrus API URL")] = HYDRUS_API_URL,
     overwrite: Annotated[Optional[bool], typer.Option(help="Overwrite existing perceptual hashes")] = False,
     query: Annotated[Optional[List[str]], typer.Option(help="Custom Hydrus tag query")] = HYDRUS_QUERY,
+    file_service_key: Annotated[
```

Comment:
@appleappleapplenanner Just curious - since you went ahead and added support for multiple file service keys, do you want me to change this argument to be a List[str], so that folks can specify multiple keys, or did you intentionally leave it at a just supporting a single key?

Reply:
You know what, the entire file_service_keys should be handled the same way as queries. Literally the exact same way:
  - The list can be passed as a JSON array, file_services should be passed as a parameter to HydrusVideoDeduplicator.deduplicate()
  - Handling file_service_keys should be moved from the constructor to deduplicate(). Checking for None and falling back to all_local_files included
  - Whatever else I did with queries, do it in a similar way

deduplicate is called at the bottom of dedup.py

Even if this is not a good way to do things, it will at least be consistent.

Why did I not think of this before? If you don't want to do this I'll do it.

Reply:
@appleappleapplenanner It is done!